projects/addon-commerce/components/input-card/input-card.options.ts

```diff
@@ -18,22 +19,10 @@ export interface TuiInputCardOptions {
     >;
 }
 
-// TODO: Move payment system icons into its own token in 4.0
-export const TUI_INPUT_CARD_DEFAULT_OPTIONS: TuiInputCardOptions = {
-    icons: TUI_PAYMENT_SYSTEM_ICONS,
-    cardSrc: '',
-    paymentSystemHandler: tuiGetPaymentSystem,
-    autocompleteEnabled: false,
-};
-
-export const TUI_INPUT_CARD_OPTIONS = tuiCreateToken(TUI_INPUT_CARD_DEFAULT_OPTIONS);
-
-export function tuiInputCardOptionsProvider(
-    options: Partial<TuiInputCardOptions>,
-): Provider {
-    return tuiProvideOptions(
-        TUI_INPUT_CARD_OPTIONS,
-        options,
-        TUI_INPUT_CARD_DEFAULT_OPTIONS,
-    );
-}
+export const TUI_INPUT_CARD_OPTIONS: InjectionToken<TuiInputCardOptions> =
+    tuiCreateTokenFromFactory(() => ({
+        cardSrc: '',
+        autocompleteEnabled: false,
+        paymentSystemHandler: tuiGetPaymentSystem,
+        icons: inject(TUI_PAYMENT_SYSTEM_ICONS_OPTIONS),
```

Comment:
No point in duplicating it in another token

Reply:
fixed